### ISSUE TITLE
publishcv.sh assumes that hammer content-view info lists CV versions in chronological order

### ISSUE THREAD

**Issue by pcfe** [RedHatSatellite/soe-ci#60]
In Sat 6.2.6 I just noticed that CV versions are not necessarily listed in chronological order.

This breaks publishcv.sh ([lines 39 through 42](https://github.com/RedHatSatellite/soe-ci/blob/master/publishcv.sh#L39-L42))

Example
```
# hammer content-view info --name "cv-soe-testing-6Server" --organization-id 1 | sed -n "/Versions:/,/Components:/p"
Versions:
 1) ID:        744
    Version:   74.0
    Published: 2017/01/10 13:15:34
2) ID:        739
    Version:   72.0
    Published: 2017/01/10 07:27:22
3) ID:        742
    Version:   73.0
    Published: 2017/01/10 12:10:11
4) ID:        696
    Version:   61.0
    Published: 2017/01/04 12:50:57
5) ID:        697
    Version:   62.0
    Published: 2017/01/04 15:53:22
6) ID:        700
    Version:   63.0
    Published: 2017/01/04 16:40:46
7) ID:        701
    Version:   64.0
    Published: 2017/01/04 17:30:44
8) ID:        703
    Version:   65.0
    Published: 2017/01/04 18:37:44
9) ID:        704
    Version:   66.0
    Published: 2017/01/04 19:14:22
10)ID:        705
    Version:   67.0
    Published: 2017/01/05 11:08:41
11)ID:        708
    Version:   68.0
    Published: 2017/01/05 13:06:46
12)ID:        709
    Version:   69.0
    Published: 2017/01/05 14:24:48
13)ID:        711
    Version:   70.0
    Published: 2017/01/05 15:25:52
14)ID:        728
    Version:   71.0
    Published: 2017/01/09 18:27:05
Components:
```

**Comment by pcfe** [comment]
commit b6607c4348275ea2efd00a643c6541dad4c90ef2 should fix this.
Still need to test though.

**Comment by pcfe** [comment]
fixed by d11ac62
